HW2/index.css

```diff
@@ -0,0 +1,260 @@
+* {
+	padding: 0;
+	margin: 0;
+	box-sizing: border-box;
+}
+body {
+	width: 1366px;
+	height: 950px;
+	background-color: #f9f9f9;
+}
+.wrapper {
+	width: 80%;
+	margin: 0 auto;
+	background-color: #fff;
+	box-shadow: 0 0 10px rgba(0,0,0,0.5);
+	padding: 50px;
+}
+/****************HEADER**************/
+.header {
+	width: 100%;
+}
+.logo {
+	color: #323232;
+	font-size: 40px;
+	font-family: System;
+	font-weight: bold;
+}
+.menu {
+	background: -moz-linear-gradient(45deg, rgba(69,68,68,1) 0%, rgba(94,94,94,1) 100%); /* ff3.6+ */
+	background: -webkit-gradient(linear, left bottom, right top, color-stop(0%, rgba(69,68,68,1)), color-stop(100%, rgba(94,94,94,1))); /* safari4+,chrome */
+	background: -webkit-linear-gradient(45deg, rgba(69,68,68,1) 0%, rgba(94,94,94,1) 100%); /* safari5.1+,chrome10+ */
+    background: -o-linear-gradient(45deg, rgba(69,68,68,1) 0%, rgba(94,94,94,1) 100%); /* opera 11.10+ */
+    background: -ms-linear-gradient(45deg, rgba(69,68,68,1) 0%, rgba(94,94,94,1) 100%); /* ie10+ */
+    background: linear-gradient(45deg, rgba(69,68,68,1) 0%, rgba(94,94,94,1) 100%); /* w3c */
+    filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#5e5e5e', endColorstr='#454444',GradientType=1 ); /* ie6-9 */
+	margin-top: 40px;
+	display: flex;
+	background-color: #ffffff;
+    background-image: linear-gradient(to top, #444343 0%, #5e5e5e 100%);
+    list-style: none;
+
+}
+
+.menu__link {
+	text-decoration: none;
+	color: #fff;
+	font-size: 15px;
+	display: block;
+	height: 40px;	
+	padding-top: 10px;
+	padding-left: 25px;
+	padding-right: 25px;
+	margin-right: 15px;
+	margin-left: 10px;
+	text-shadow: 1px 1px 1px rgba(0, 0, 0, 0.5);
+}
+.menu__link:first-child {
+	margin-left: 0px;
+}
+.menu__link:hover {
+	background-color: #353535;
+	border-right: 1px solid #838383;
+	border-left: 1px solid #838383;
+	margin-right: 14px;
+	margin-left: 9px;
+}
+.menu__link:first-child:hover {
+	margin-left: -1px;
+}
+
+/****************CONTENT***********/
+.container {
+	width: 100%;
+	display: flex;
+	justify-content: space-between;
+	margin-top: 40px;
+	background-color: #ffffff; 
+}
+.aside {
+	width: 25%;
+}
+.article {
+	width: 70%;
+}
+.aside li:last-child {
+	border-bottom: 1px solid #c8c8c8;
+}
+.aside li{
+	list-style: none;
+	border-top: 1px solid #c8c8c8;
+	border-left: 1px solid #c8c8c8;
+	border-right: 1px solid #c8c8c8;
+	font-family: "2211.ttf";
+}
+.aside__link {
+	font-size: 15px;
+	text-decoration: none;
+	color: #656363;
+	height: 40px;
+	padding: 10px 24px;
+	display: block;			
+}
+.aside__link:hover {
+	color: #2c2c2c;
+	background-color: #f9f9f9;
+	padding-left: 40px;
+
+}
+.article_picture {
+	border: 1px solid #c8c8c8;
+	width: 100%;
+	display: flex;
+	width: 694px;
```

Comment:
don't use fixed sizes for pictures, here could be another picture and it will look awful